tools/uf2conv.py

```diff
@@ -347,8 +368,17 @@ def error(msg):
             try:
                 print("Resetting " + str(args.serial))
                 try:
-                    ser = serial.Serial(args.serial, 1200)
+                    ser = serial.Serial()
+                    ser.port = args.serial
+                    ser.open()
+                    ser.baudrate = 9600
```

Comment:
There are no bootloaders on the Pico (or in the Arduino ecosystem AFAIK) which use a tickle at 9600bps to force a reset.  Why are you doing this?

Also, RTS is not examined in the code which does resets on the Pico, so there is no reason to set it high or low.

Reply:
Done so the TTY is in a known state, thus you are assured the transition to 1200 is seen.

FWIW using `stty -F /dev/ttyACM0 1200` works like this too.
If the kernel thinks the baud rate is already something that you request, it won't send it along.
This simply ensures that it works, and to be honest has improved successful reboot to bootloader for me, and can't hurt for those who didn't have the problem.